plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/store/LogEvent.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2008-2015 Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+/*global Ext*/
+
+/**
+ * LogEvent store.
+ *
+ * @since 3.0
+ */
+Ext.define('NX.store.LogEvent', {
+  extend: 'Ext.data.Store',
+  model: 'NX.model.LogEvent'
```

Comment:
agree, though I'd still like to learn how to properly scroll, we kinda do that for the logging viewer...